### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S .eggs,po,*.css -L boddy,maco,trough,wont,poing,gir,visibles`

Let me know if you want to add this to the GitHub CI job.

### CODE REVIEW

File: terminatorlib/notebook.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def replace(self, oldwidget, newwidget):
 
     def get_child_metadata(self, widget):
         """Fetch the relevant metadata for a widget which we'd need
-        to recreate it when it's readded"""
```
Comment (Vulcalien) [inline terminatorlib/notebook.py:242]: This changes meaning, which is correct?

Comment (kianmeng) [reply terminatorlib/notebook.py:242]: Updated.

**Review by Vulcalien — APPROVED** [review]

File: terminatorlib/notebook.py (hunk 1 above)
Comment (mattrose) [reply terminatorlib/notebook.py:242]: this looks good

**Comment by mattrose** [comment]
Thanks for fixing this, and Thanks for your once-over on this @Vulcalien

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**